react/src/js/test/unit/react/friends/FriendsStoreTests.js

```diff
@@ -0,0 +1,94 @@
+import {expect} from "chai";
+import sinon from "sinon";
+
+import FriendsStore from "react/friends/FriendsStore";
+import RestUtil from "react/util/RestUtil";
+
+import {createMockPromise} from "../../util/PromiseUtil";
+
+const MOCK_USER_INFO = {
+    id: "abc-defg-hijkl"
+};
+
+const MOCK_REQUEST = {
+    id: "http://127.0.0.1/author/1",
+    displayName: "Request 1",
+    host: "http://127.0.0.1",
+    url: "http://127.0.0.1/author/1"
```

Comment:
you changed all the urls above to be :8000, do you want to do the same here for consistency?

Reply:
No, that was only on the backend. The tests here don't communicate with the backend so that isn't necessary